src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -349,74 +377,83 @@ export const PositionsTable = ({ positions, displayLine = false }: PositionsProp
 
   return (
     <>
-      <TableHeader>{t('components.positions-card.positions')}</TableHeader>
-      <ViewContainer>
+      {/* <TableHeader>{t('components.positions-card.positions')}</TableHeader> */}
```

Comment:
I thought u intentionally got rid of the title inside the Position+Collateral Tables  in a previous PR to use the `TableHeader` component exclusively no?

Reply:
I did, but as I made the style changes in this PR, it ended up looking better the way it was originally.